src/averageAges.js

```diff
@@ -15,11 +15,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const findMen = century ? people.filter(
+    men => (Math.ceil(men.died / 100)) === century && men.sex === 'm'
+  ) : people.filter(men => men.sex === 'm');
```

Comment:
```suggestion
  const findMen = century 
    ? people.filter(
      men => (Math.ceil(men.died / 100)) === century && men.sex === 'm'
     ) 
    : people.filter(men => men.sex === 'm');
```
Btw, you do filter in both cases, so you can just implement different logic inside filter callback